src/uproot/models/RNTuple.py

```diff
@@ -218,26 +220,62 @@ def page_list_envelopes(self):
 
         return self.column_innerlist_dict
 
-    def col_form(self, field_id):
-        for cr in self.header.column_records:
+    def base_col_form(self, col_id):
+        cr = self.header.column_records[col_id]
+        form_key = f"column-{col_id}"
+        if cr.type == 3:  # switch (UnionForm)
```

Comment:
This "`3`" and all others like it (e.g. the "`2`" a couple of lines down) should all be constants defined in `uproot.const`. They should have a common prefix to indicate that they're for RNTuple.